src/SyncGate.tsx

```diff
@@ -95,6 +105,46 @@ const syncInfoSelector = createSelector(
       },
       Map<string, SyncInfoJournal>()
     );
+
+    const bdayCollection = new EteSync.CollectionInfo();
+    bdayCollection.uid = "birthdays"; 
+    bdayCollection.type = "CALENDAR";
+    bdayCollection.displayName = "Birthdays";
+    bdayCollection.color = -1413414;
+
+    const bdayJournal = new EteSync.Journal({ uid: bdayCollection.uid });
+    const cryptoManager = new EteSync.CryptoManager(etesync.encryptionKey, bdayCollection.uid);
+    bdayJournal.setInfo(cryptoManager, bdayCollection);
+
+    let bdayEntries = List<EteSync.SyncEntry>();
+
+    Object.values(addressBookItems).filter((c) => c.bday).forEach((c) => {
+      const bdayTime = parseDate(c.comp.getFirstProperty("bday"));
+      if (bdayTime === {} || bdayTime.month === undefined) {
+        return;
+      }
+      const year = bdayTime.year ?? 1900;
+      const month = (bdayTime.month + 1).toString().padStart(2, "0");
+      const day = bdayTime.day.toString().padStart(2, "0");
+
+      const event =
+        "BEGIN:VCALENDAR\nBEGIN:VEVENT\n" +
+        `SUMMARY:${c.fn}'s Birthday\n` +
+        `UID:${uuid.v4()}\n` +
```

Comment:
Just a thought (maybe a bad idea), but wouldn't it be better to use the contact's uid here instead of a randomly generated one? This will make it persistent across refreshes.

Reply:
I rearranged the code a bit to allow the uid to be a combination of the journal uid and contact uid. This should allow persistence and avoid collisions.

Reply:
Great!